### ISSUE TITLE
Is it just me or does it never suggest multiline completions?

### ISSUE THREAD

**Issue by IndianBoy42** [Exafunction/codeium.nvim#186]
Is there any way to request or bias it towards longer snippets?

**Comment by bruteforks** [comment]
yeah this is the worst part about trying all of these plugins. Haven't found one that does multi-line *with* completion. They do have some that you add a comment, and the LLM gives back a multi-line answer though.
